content/developer/api/version-8/JSON-API.adoc

```diff
@@ -321,14 +321,12 @@ DELETE {{your-SuiteCRM-url}}/V8/module/{moduleName}/{id}
 === Get relationship
 
 [source,php]
-GET {{your-SuiteCRM-url}}/V8/module/{moduleName}/relationships/{relation}
-
-Available parameters: fields, page, sort, filter
+GET {{your-SuiteCRM-url}}/V8/module/{moduleName}/{id}/relationships/{relation}
```

Comment:
I'd maybe use `linkName` or similar instead of `relation` as it is the link name for the relationship that needs to be passed not the relationship name as these may differ.